package.json

```diff
@@ -1,18 +1,13 @@
 {
-  "name": "mytunnel",
+  "name": "fusetunnel",
```

Comment:
Package should be @fusebit/tunnel (probably).

Reply:
Done